### PR TITLE
[ACS-7295] create actions - beforeEach and afterAll refactored

### PR DESCRIPTION
**Pull request by datguychen**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://hyland.atlassian.net/browse/ACS-7295

**What is the new behaviour?**
[ACS-7295] create actions - beforeEach and afterAll refactored

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

[ACS-7295]: https://hyland.atlassian.net/browse/ACS-7295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by katarzynakita — APPROVED** [review]

File: e2e/playwright/create-actions/src/tests/create-library.spec.ts [hunk 1]
```diff
@@ -79,29 +84,13 @@ test.describe('Create Libraries ', () => {
...
-      console.error(`beforeEach failed : ${error}`);
-    }
+    await Utils.tryLoginUser(loginPage, username, username, 'Main beforeEach failed');
+    await myLibrariesPage.navigate();
+    await myLibrariesPage.selectCreateLibrary();
   });
 
   test.afterAll(async () => {
-    try {
-      await sitesApi.deleteSites(createdLibrariesIds);
```
Comment (nikita-web-ua) [inline e2e/playwright/create-actions/src/tests/create-library.spec.ts:99]: `createdLibrariesIds[]` is not used anymore? can be removed?

Comment (datguychen) [reply e2e/playwright/create-actions/src/tests/create-library.spec.ts:99]: My bad, fixed it. Now this array is added to the Utils method.

**Review by nikita-web-ua — APPROVED** [review]

File: e2e/playwright/copy-move-actions/src/tests/move.test.ts [hunk 2]
```diff
@@ -68,6 +69,10 @@ test.describe('Move actions', () => {
     await personalFiles.navigate();
   });
 
+  test.afterAll(async () => {
```
Comment (akashrathod28) [inline e2e/playwright/copy-move-actions/src/tests/move.test.ts:72]: can you please check/remove afterAll written at line 42

Comment (datguychen) [reply e2e/playwright/copy-move-actions/src/tests/move.test.ts:72]: Oh I didnt see its already there...

**Pull request merged**